packages/features/ee/workflows/components/WorkflowStepContainer.tsx

```diff
@@ -7,6 +7,7 @@ import type { UseFormReturn } from "react-hook-form";
 import { Controller } from "react-hook-form";
 import "react-phone-number-input/style.css";
 
+import type { RetellAgentWithDetails } from "@calcom/features/calAIPhone/providers/retellAI";
```

Comment:
![Image](https://github.com/user-attachments/assets/4b50858e-81e3-4b22-9e4e-c12ff23cec5e)

button should be destructive